### ISSUE TITLE
Settings window open when system starts

### ISSUE THREAD

**Issue by stepanp** [deseven/icanhazshortcut#15]

**Comment by deseven** [comment]
Can't reproduce it.
What are your MacOS and iCHS versions?

**Comment by stepanp** [comment]
macOS el captain 10.11.4, iCHS latest

**Comment by deseven** [comment]
I'm currently on 10.12, but not so long ago i had 10.11. Besides, i have 10.8 in a VM, also works fine there.
Do you have any system-wide tweaks which may alter windows and apps behavior?

**Comment by stepanp** [comment]
No, my system hasn't tweaks. So, this bug reproduces not every time.
I'm try to reproduce now and all works fine.

**Comment by deseven** [comment]
Ok, let's leave this issue open for now, in the next version i'll try to do some workarounds to make sure that settings window is closed on program start.

**Comment by ghost** [comment]
FYI, I've been on El Capitan for a long time (currently on 10.11.6) and can't reproduce this issue either.
Maybe the settings window was already left open before rebooting and it was just out of sight (due to using Spaces or a Single Application Mode app).

**Comment by deseven** [comment]
Thanks, i'm closing this for now.

**Comment by deseven** [comment]
I actually finally got it! Don't really understand why it happens and it's really hard to debug since it happens very rarely, but it's certainly an issue.

**Comment by FurloSK** [comment]
Yes, this sporadically happens to me on my system too (macOS 10.14.6 Mojave, iCHS 1.3.0.

I tried turning it off an on again; and also turning it off and manually adding the app to login items through system preferences (with "hidden" checked) – neither helped. _Sometimes_, it just shows the settings window.

**Comment by deseven** [comment]
I remember spending hours and hours trying to figure it out, putting breakpoints and debug lines everywhere, but the real cause was never clear.

In any case, this no longer should be a problem for the v2.

**Comment by deseven** [comment]
Fixed in version 2.0.0, pre-release is available already if you'd like to help with the testing:  
https://github.com/deseven/iCanHazShortcut/releases/tag/2.0.0-preview
